backend/src/routes/room-routes.ts

```diff
@@ -0,0 +1,13 @@
+import { Express } from "express";
+
+export default function roomRoutes(app: Express, rooms: { [key: string]: any }) {
+  app.get('/rooms', (_req, res) => {
+    res.send(Object.keys(rooms));
+  });
+
+  app.get('/room/:id', (req, res) => {
+    const room = req.params.id;
+    res.send({ players: rooms[room].players });
+  });
+
+}
```

Comment:
suggestion (part 2): if we want to use the express router here, then this would look something like this: 
```suggestion

export const roomRouter = express.Router()

  roomRouter.get('/rooms', (_req, res) => {
    res.send(Object.keys(rooms));
  });

  roomRouter.get('/room/:id', (req, res) => {
    const room = req.params.id;
    res.send({ players: rooms[room].players });
  });

```